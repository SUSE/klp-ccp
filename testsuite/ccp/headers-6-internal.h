void
#include "headers-6-1.h"
(void)
#include "headers-6-2.h"
}
