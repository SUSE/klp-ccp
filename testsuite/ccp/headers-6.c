static
#include "headers-6-internal.h"

void pu_f(void)
{
	g();
}
