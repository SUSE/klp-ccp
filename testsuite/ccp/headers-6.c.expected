/* klp-ccp: from testsuite/ccp/headers-6.c */
static
/* klp-ccp: from testsuite/ccp/headers-6-internal.h */
void
#include "headers-6-1.h"
(void)
#include "headers-6-2.h"
}

/* klp-ccp: from testsuite/ccp/headers-6.c */
void klpp_pu_f(void)
{
	g();
}
